.rubocop_todo.yml

```diff
@@ -0,0 +1,25 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2021-11-03 05:15:19 UTC using RuboCop version 1.22.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 2
+# Configuration parameters: AssignmentOnly.
+RSpec/InstanceVariable:
```

Comment:
I think we can disable this cop at all, in `.rubocop.yml` since we will making test files with @ usage